algorithms/ar-lscsa/python3/max_sum_array.py

```diff
@@ -0,0 +1,8 @@
+def maxSubArray(nums):
+       num_array = [0 for i in range(len(nums))]
+       num_array[0] = nums[0]
+       for i in range(1,len(nums)):
+              num_array[i] = max(num_array[i-1]+nums[i],nums[i])
+       return max(num_array)
+nums = [-13,-3,-25,-20,2,1,-3,7,-2,2,1,-5,-3,-16] 
```

Comment:
Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396358)

Reply:
@LON3WO7F 
There a space at the end of line 7, remove it, it will fix this issue